data/institutions/ville_champagne_au_mont_d_or.yml

```
@@ -0,0 +1,5 @@
+name: Ville de Champagne au Mont d'Or
+imgSrc: img/institutions/logo_ville_champagne_au_mont_d_or.svg.png
```

Petit coquille dans le nom. On peut également supprimer l'autre logo de la ville vu qu'il n'est pas utilisé
 
```suggestion
imgSrc: img/institutions/logo_ville_champagne_au_mont_d_or.png
```

---

J'ai une petite question, pourquoi ne pas utiliser le fichier svg d'origine ?